guides/common/modules/proc_calling-the-api-in-python.adoc

```diff
@@ -82,23 +82,23 @@ def main():
     """
 
     # Check if our organization already exists
-    org = get_json(f"{project_allcaps}_API/organizations/{ORG_NAME}")
+    org = get_json(f"{project-allcaps}_API/organizations/\{ORG_NAME}")
 
     # If our organization is not found, create it
     if org.get('error', None):
         org_id = post_json(
-            f"{project_allcaps}_API/organizations/",
+            f"\{{project-allcaps}_API}/organizations/",
```

Comment:
why is this rendered as:
```
f"\{FOREMAN_API}/organizations/",
```
Is the `\` too much

Reply:
It was much easier that I thought, it really was just the one backslash :see_no_evil: